### PR TITLE
Add Firestore IN snippets

### PR DESCRIPTION
**Pull request by morganchen12**
TODO: Add snippets to snippet registry, remove `master` from Podfiles (or don't)

Note that iOS has one less snippet than Android since there's only one method (barring FieldPath) for creating `arrayContainsAny` queries.

### CODE REVIEW

**Comment by samtstern** [comment]
I think you're missing this snippet:
https://github.com/firebase/snippets-android/pull/145/files#diff-90154b91442e0aa843eff0c1bd13c345R1052

**Comment by morganchen12** [comment]
I may be missing some context here--the only two `IN` methods on iOS are declared [here](https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/Source/Public/FIRQuery.h#L281-L309) and there's no variadic method, using an array is the only option. 

```objc
- (FIRQuery *)queryWhereField:(NSString *)field
                           in:(NSArray<id> *)value NS_SWIFT_NAME(whereField(_:in:));

- (FIRQuery *)queryWhereFieldPath:(FIRFieldPath *)path
                               in:(NSArray<id> *)value NS_SWIFT_NAME(whereField(_:in:));
```

Should the missing snippet just be translated to this?

```swift
citiesRef.where("regions", in: ["west_coast", "east_coast"])
```

**Comment by samtstern** [comment]
@morganchen12 if you look at Juan's docs CL it's clearer.  He wants to show what happens if you do `in` on a field that is an array.

**Comment by morganchen12** [comment]
I don't think I'm cc'd on that CL. Can you add me?

**Comment by morganchen12** [comment]
Fixed the missing snippet, thanks @samtstern!

File: firestore/objc/Podfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ target 'firestore-smoketest-objc' do
   # Pods for firestore-smoketest
   pod 'Firebase/Core'
   pod 'Firebase/Auth'
-  pod 'Firebase/Firestore'
+  pod 'FirebaseFirestore', :git => 'https://github.com/firebase/firebase-ios-sdk.git', :branch => 'master'
```
Comment (samtstern) [inline firestore/objc/Podfile]: Just watch out for this (and other references to master git)

**Pull request merged**